### DISCUSSION TITLE
Request for backporting fix for CVE-2024-32866 to version 0.9.x

### DISCUSSION THREAD

**Discussion by vtsvetkov-splunk** [edmundhung/conform#641] [General]
Hi @edmundhung,

Firstly, thank you for your hard work on the conform library. I liked it so much that I decided to adopt it before a stable version was even released. However, I ran into trouble when a vulnerability ([CVE-2024-32866](https://github.com/advisories/GHSA-624g-8qjg-8qxf)) was discovered in versions prior to 1.1.1.

I am using version 0.9.1 because I have React 16, and that's the latest version with React 16 support. Due to other corporate dependencies that rely on React 16, I can't upgrade to a newer version yet. A security audit flagged this version, and I need to address the vulnerability.

Is it feasible to backport the fix for this vulnerability to version 0.9?

Thank you,
Viktor

**Comment by edmundhung** [comment]
Hi @vtsvetkov-splunk, you can now upgrade to v0.9.2 with a patch for the vulnerability issue :)

**Comment by vtsvetkov-splunk** [comment]
Thank you very much!
